modules/ml/src/test/java/org/apache/ignite/ml/regressions/linear/ArtificialRegressionDatasets.java

```diff
@@ -0,0 +1,404 @@
+            -2.13208328824, 0.381223328983, 0.283479210749, 1.17705098922, -2.38800904207, 1.32108350152},
+        {-148.479593311, -0.814604260049, -0.821204361946, -1.08768677334, -0.0659445766599,
+            0.583741297405, 0.669345853296, -0.0935352010726, -0.254906787938, -0.394599725657, -1.26305927257},
+        {244.865845084, 0.776784257443, 0.267205388558, 2.37746488031, -0.379275360853,
+            -0.157454754411, -0.359580726073, 0.886887721861, 1.53707627973, 0.634390546684, 0.984864824122},
+        {-81.9954096721, 0.594841146008, -1.22273253129, 0.532466794358, 1.69864239257,
+            -0.12293671327, -2.06645974171, 0.611808231703, -1.32291985291, 0.722066660478, -0.0021343848511},
+        {-245.715046329, -1.77850303496, -0.176518810079, 1.20463434525, -0.597826204963,
+            -1.45842350123, -0.765730251727, -2.17764204443, 0.12996635702, -0.705509516482, 0.170639846082},
+        {123.011946043, -0.909707162714, 0.92357208515, 0.373251929121, 1.24629576577,
+            0.0662688299998, -0.372240547929, -0.739353735168, 0.323495756066, 0.954154005738, 0.69606859977},
+        {-70.4564963177, 0.650682297051, 0.378131376232, 1.37860253614, -0.924042783872,
+            0.802851073842, -0.450299927542, 0.235646185302, -0.148779896161, 1.01308126122, -0.48206889502},
+        {21.5288687935, 0.290876355386, 0.0765702960599, 0.905225489744, 0.252841861521,
+            1.26729272819, 0.315397441908, -2.00317261368, -0.250990653758, 0.425615332405, 0.0875320802483},
+        {231.370169905, 0.535138021352, -1.07151617232, 0.824383756287, 1.84428896701,
+            -0.890892034494, 0.0480296332924, -0.59251208055, 0.267564961845, -0.230698441998, 0.857077278291},
+        {38.8318274023, 2.63547217711, -0.585553060394, 0.430550920323, -0.532619160993,
+            1.25335488136, -1.65265278435, 0.0433880112291, -0.166143379872, 0.534066441314, 1.18929937797},
+        {116.362219013, -0.275949982433, 0.468069787645, -0.879814121059, 0.862799331322,
+            1.18464846725, 0.747084253268, 1.39202500691, -1.23374181275, 0.0949815110503, 0.696546907194},
+        {260.540154731, 1.13798788241, -0.0991903174656, 0.1241636043, -0.201415073037,
+            1.57683389508, 1.81535629587, 1.07873616646, -0.355800782882, 2.18333193195, 0.0711071144615},
+        {-165.835194521, -2.76613178307, 0.805314338858, 0.81526046683, -0.710489036197,
+            -1.20189542317, -0.692110074722, -0.117239516622, 1.0431459458, -0.111898596299, -0.0775811519297},
+        {-341.189958588, 0.668555635008, -1.0940034941, -0.497881262778, -0.603682823779,
+            -0.396875163796, -0.849144848521, 0.403936807183, -1.82076277475, -0.137500972546, -1.22769896568}
+    }, new double[] {45.8685095528, 11.9400336005, 16.3984976652, 79.9069814034, 5.65486853464,
+        83.6427296424, 27.4571268153, 73.5881193584, 27.1465364511, 79.4095449062}, -5.14077007134);
+
+    /** */
+    public static class Dataset {
```

Comment:
Please rename this class to TestDataset or something else. Or use LabeledDataset after update

Reply:
Renamed.